### ISSUE TITLE
HELPME_GITHUB_TOKEN 

### ISSUE THREAD

**Issue by rougier** [vsoch/helpme#15]
I'm trying to test the software. Everything went smooth but when I tried to post an issue here (`helpme github vsoch/helpme`), it complains about:

```
ERROR Cannot create issue. Does your token have scope repo?
```

I created a token via GiHub with public repo permissions but it does not seem to be sufficient. Can you detail what is needed exactly (minimum settings)?

**Comment by vsoch** [comment]
Yeah! When you create the token, you have to define it's scope (what it's allowed to do in using the API). For example, here is the full scopes that I've defined for the token I created for helpme-client:

![image](https://user-images.githubusercontent.com/814322/41391083-85a9e080-6f4d-11e8-9519-459fa151d0a6.png)

Can you show me what you created and we can go from there?

**Comment by rougier** [comment]
I allowed all the repos but the full control. Is that one needed too ?
gist & notification not really needed I guess, no?

**Comment by vsoch** [comment]
I think to have write permission to create issues, full control would be needed. Likely gist aren't, and probably not notification (but I think I was testing using it in case I wanted to add features!)

**Comment by rougier** [comment]
No luck so far. I'll create a token with every permission and see if this works.

**Comment by rougier** [comment]
Not working, same message. What I did:

1. Create a token via GitHub with all permission
2. Run docker
3. `export HELPME_GITHUB_TOKEN=xxxxxxxxxxxxxx`
4. `helpme github vsoch/helpme`

**Comment by vsoch** [comment]
Hey and just a heads up that I'm going off to bed, but I'll be able to check and help again tomorrow morning!

**Comment by rougier** [comment]
Good night.

Here is my session:

```
(base) root@2ae4de1339a3:~# export HELPME_GITHUB_TOKEN=xxxxxxx...
(base) root@2ae4de1339a3:~# helpme github vsoch/helpme
[helpme|github]
Hello Friend! I'll help you post a Github issue today.
Please describe the issue you are having: Testing helpme for review
Would you like to send a terminal recording?
Please enter your choice [Y/N/y/n] : n
Environment  USER|TERM|SHELL|PATH|LD_LIBRARY_PATH|PWD|JAVA_HOME|LANG|HOME|DISPLAY
Is this list ok to share?
Please enter your choice [Y/N/y/n] : y
ERROR Cannot create issue. Does your token have scope repo?
(base) root@2ae4de1339a3:~# helpme github vsoch/helpme
[helpme|github]
Hello Friend! I'll help you post a Github issue today.
Please describe the issue you are having: test
Would you like to send a terminal recording?
Please enter your choice [Y/N/y/n] : n
Environment  USER|TERM|SHELL|PATH|LD_LIBRARY_PATH|PWD|JAVA_HOME|LANG|HOME|DISPLAY
Is this list ok to share?
Please enter your choice [Y/N/y/n] : y
ERROR Cannot create issue. Does your token have scope repo?
```

Most probably, this is a owner permission problem. I create a fake repo and I was able to post an issue from the command line, see https://github.com/rougier/helpme/issues/1

You will need to add some documentation about that in the README. But this is annoying if I cannot post an issue to a repo I do not own.

**Comment by rougier** [comment]
And now you can try `helpme github rougier/helpme` to check if it is working for you.

**Comment by vsoch** [comment]
haha I think I just got this working -> https://github.com/rougier/helpme/issues/2

The fixes are merged to master, and the updated package on pypi. The Docker image is not done yet. you can check on the status here if you want to test with it --> https://hub.docker.com/r/vanessa/helpme/builds/

TLDR: the API is different. Maybe Microsoft is responsible... :P

**Comment by rougier** [comment]
Ok, ping me when you've update the docker and I will test on my side.
Did you experience same problem as me before the fix?

**Comment by vsoch** [comment]
Yes I did! It was an oversight on my part to not check that repos with other ownership didn't work. The fix then was to use a different (correct) endpoint, for some reason I had chosen the import endpoint (which of course would require owner / admin permissions on the issue board) instead of vanilla issue --> create. Please give it another try and let me know if it works for you!
